scripts/generate_big_data_network.py

```diff
@@ -0,0 +1,358 @@
+
+        units = self.get_current_network()
+        if len(units) == 0:
+            print("Simulating new data network from {} units over {} nights. This make take "
+                  "a few minutes...".format(num_units, num_nights), file=sys.stdout)
+            units = self.init_units(num_units)
+        else:
+            num_units = len(units)
+            self.update_cameras()
+            print('Adding new simulated data to current network of {} units over {} nights. This may '
+                  'take a few minutes...'.format(num_units, num_nights), file=sys.stdout)
+
+        temp_dir = '/tmp/sim-data'
+
+        # For every night, let every unit observe a few stars, and
+        # output a Postage Stamp Cube (PSC) and light curve for each star.
+        lc_count = 0
+        curr_date = start_date
+        while curr_date <= end_date:
+            for unit in units:
+
+                # Start first sequence 1-2 hours after sunset
+                site = self.unit_dict[unit]
+                loc = Observer.at_site(site)
+                sunset = loc.sun_set_time(
+                    Time(curr_date, scale='utc'), which='next')
+                sunset_time = datetime.strptime(
+                    sunset.isot, "%Y-%m-%dT%H:%M:%S.%f")
+                last_obs_end_time = sunset_time + \
+                    timedelta(minutes=(random.random() * 60 + 60))
+                sequences_per_night = random.randint(4, 6)
+                for i in range(sequences_per_night):
+
+                    stars_per_sequence = random.randint(100, 200)
+                    start_time, end_time = self.set_obs_time(last_obs_end_time)
+                    last_obs_end_time = end_time
+
+                    for j in range(stars_per_sequence):
+
+                        # Get observation information
+                        field = self.get_field()
+                        pic, coords = self.get_fake_pic()
+
+                        for camera in self.cameras[unit]:
+                            psc_filename = "PSC/{}/{}/{}/{}.fits".format(
+                                unit, camera, start_time.isoformat(), pic)
+                            lc_filename = "LC/{}/{}/{}/{}.json".format(pic, unit, camera,
+                                                                       start_time.isoformat())
+
+                            # Create data products
+                            hdu = self.build_psc(unit, camera, field,
+                                                 pic, coords, start_time, end_time)
+                            lc = self.build_lightcurve(hdu)
+
+                            # Write data products to local temp files
+                            self.write_psc(
+                                "{}/{}".format(temp_dir, psc_filename), hdu)
+                            self.write_lightcurve(
+                                "{}/{}".format(temp_dir, lc_filename), lc)
+
+                            # Upload data products from local files to cloud
```

Comment:
not unrealistic, that's what i've been doing - right now it always downloads the files to local temp storage and then uploads them to cloud. though it never clears the temp dir, which should happen once in a while..

it shouldn't be hard to add a command parameter to decide whether to store locally or upload to cloud. only caveat: if just want to upload to cloud without first downloading locally, would need to add a new function to the storage util in POCS, blob's [upload_from_file](https://googlecloudplatform.github.io/gcloud-python/latest/storage-blobs.html) method. currently only uses upload_from_string (which can be used for light curve JSONs but not PSCs) or upload_from_filename. 

i'll quickly add a toggle and option for where to store files locally now.